Code/GraphMol/Depictor/Wrap/testDepictor.py

```diff
@@ -559,5 +559,167 @@ def testUsingCoordGenCtxtMgr(self):
+        template_match_center = sum(template_match_positions) / len(template_match_positions)
+        mol_positions = [p - template_match_center for p in mol.GetConformer().GetPositions()]
+
+        template_center = sum(template.GetConformer().GetPositions()) / template.GetNumAtoms()
+        template_positions = [p - template_center for p in template.GetConformer().GetPositions()]
+
+        # the mol may match the template but be slightly rotated about the centroid
+        # or reflected across the x or y axis
+        rotations = [[], [], [], []]
+        for template_idx, idx in enumerate(match):
+            v1 = mol_positions[idx]
+
+            # no reflection
+            v2 = template_positions[template_idx]
+            val = round(np.dot(v1, v2) / (np.linalg.norm(v1) * np.linalg.norm(v2)), 4)
+            rotations[0].append(np.arccos(val))
+
+            # reflect across x-axis
+            v2[0] = v2[0] * -1
+            val = round(np.dot(v1, v2) / (np.linalg.norm(v1) * np.linalg.norm(v2)), 4)
+            rotations[1].append(np.arccos(val))
+
+            # reflect across y-axis
+            v2[0] = v2[0] * -1
+            v2[1] = v2[1] * -1
+            val = round(np.dot(v1, v2) / (np.linalg.norm(v1) * np.linalg.norm(v2)), 4)
+            rotations[2].append(np.arccos(val))
+
+
+            # reflect across y-axis and x-acis
+            v2[0] = v2[0] * -1
+            # v2[1] = v2[1] * -1
+            val = round(np.dot(v1, v2) / (np.linalg.norm(v1) * np.linalg.norm(v2)), 4)
+            rotations[3].append(np.arccos(val))
+
+        # if all the rotations are similar, then the shape is the same
+        return np.any([np.allclose(r, r[0], atol=.05) for r in rotations])
+
+    def assertMolMatchesCoordMap(self, mol, coord_map):
+        for aid, expected_position in coord_map.items():
+            actual_position = mol.GetConformer().GetAtomPosition(aid)
+            self.assertAlmostEqual(actual_position.x, expected_position.x)
+            self.assertAlmostEqual(actual_position.y, actual_position.y)
+
+
+    def testUseRingSystemTemplates(self):
+        prefer_coordgen_status = rdDepictor.GetPreferCoordGen()
+        rdDepictor.SetPreferCoordGen(False)
+
+        # test that all templates match themselves
+        fpath = os.environ['RDBASE'] + "/Data/ring_system_templates.smi"
+        with open(fpath) as f:
+            templates = [Chem.MolFromSmiles(smi) for smi in f.readlines()]
+
+        for template in templates:
+            mol = Chem.Mol(template)
+            mol.RemoveAllConformers()
+            rdDepictor.Compute2DCoords(mol)
+            assert not self.molMatchesTemplate(mol, template)
+
+            mol.RemoveAllConformers()
+            rdDepictor.Compute2DCoords(mol,useRingTemplates=True)
+            assert self.molMatchesTemplate(mol, template)
+
+        rdDepictor.SetPreferCoordGen(prefer_coordgen_status)
```

Comment:
This test no longer works since the `.smi` file has been removed.
```suggestion
```

Reply:
Totally forgot about that test!!